### PR TITLE
Burger icon functionality

### PR DESCRIPTION
**Pull request by Mahdi-Niazi**
In this milestone I:

- Add click funtionality to bars icon menu
- Add xmark icon for close the menu
- Add height to menu 
- Remove logo icon when bars clicked
- Removed envolope icon when bars clicked

### CODE REVIEW

**Comment by Mahdi-Niazi** [comment]
Inside my code editor I see no Linters error so it might be the Microverse Linters error.

File: index.html [hunk 1]
```diff
@@ -5,23 +5,27 @@
...
+        </div>
         <ul class="nav-list">
-            <li class="animation"><a href="#">Portifolio</a></li>
-            <li class="animation"><a href="#">About</a></li>
-            <li class="animation"><a href="#">Contact</a></li>
-            <li class="animation"><a href="#"><i class="fa-regular fa-envelope"></i></a></li>
+            <li class="animation nav-item"><a href="#">Portifolio</a></li>
+            <li class="animation nav-item"><a href="#">About</a></li>
+            <li class="animation nav-item"><a href="#">Contact</a></li>
+            <li class="animation nav-item envelope"><a href="#"><i class="fa-regular fa-envelope"></i></a></li>
```
Comment (mikethreels) [inline index.html]: - [ ] according to the requirements when a user clicks on the link the page should jump to the part of the page the link is referring to
- [ ] in addition to that the navigation menu should close again

File: style.css [hunk 2]
```diff
@@ -634,8 +632,9 @@ textarea::placeholder {
     background-color: #3c3a39;
   }
 
-  .nav .fa-bars {
+  .hamburger {
     display: none;
+    cursor: pointer;
```
Comment (mikethreels) [inline style.css:632]: - [ ] when hovering over a list item in the nav menu it enlarges the text however this causes the first part of the text to not be visible, this is not in line with the style guidelines, please make a change so the full text is visible also when hovering over the text.

afbeelding

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Mahdi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- good job making the navigation menu.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bienvenuushindi — APPROVED** [review]
# STATUS: APPROVED 🟢 

## Hi @Mahdi-Niazi 👋, 

Well done implementing the requested changes 👏
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Highlights 🎯 
:+1: No linter errors✔️
:+1: Good UI design✔️
:+1: Good PR title

### Optional Suggestions



**_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**